spec/classes/katello_qpid_spec.rb

```diff
@@ -7,6 +7,17 @@
         facts
       end
 
+      let :events do
```

Comment:
Ah I think I led you astray, let doesn't seem to work anywhere other than in an `it` block

From the test failure currently:

```
  `events` is not available on an example group (e.g. a `describe` or `context` block). It is only available from within individual examples (e.g. `it` blocks) or from constructs that run in the scope of an example (e.g. `before`, `let`, etc).
```